src/chocolatey/infrastructure.app/commands/ChocolateyListCommand.cs

```diff
@@ -54,6 +54,24 @@ public class ChocolateyListCommand : IListCommand<PackageResult>
             "--order-by-popularity"
         };
 
+        /// <summary>
+        /// These options have been chosen since these are the examples that were listed on docs.chocolatey.org
+        /// - choco list -li
+        /// - choco list -lai
+        /// </summary>
+        [Obsolete("Remove unsupported argument in V3!")]
+        private readonly string[] _unsupportedIncludeRegistryProgramsArguments = new[]
+        {
+            "-li",
+            "-il",
+            "-lai",
+            "-lia",
+            "-ali",
+            "-ail",
+            "-ial",
+            "-ila"

```

Comment:
I thought we were only going to include `-li`, not any other variant?

Reply:
Originally that is what I was going to do, yes, but then I saw the examples that are shown here (I mentioned this in the comment about the Obsolete attribute):

https://docs.chocolatey.org/en-us/choco/commands/list#examples

Since it was documented, I thought it best to cover that we have told folks about.